dotnet/src/SemanticKernel/Planning/FunctionViewExtensions.cs

```diff
@@ -16,7 +16,7 @@ internal static class FunctionViewExtensions
     /// <returns>A manual-friendly string for a function.</returns>
     internal static string ToManualString(this FunctionView function)
     {
-        var inputs = string.Join("\n", function.Parameters.Select(p => $"    - {p.Name}: {p.Description} (default value: {p.DefaultValue}))"));
```

Comment:
This should be resolved in `main` via #657

Reply:
There is an extra closing paranthetical - this change just removes it.

Reply:
This fixes an extra parenthesis.